### ISSUE TITLE
Faulty schema compare test

### ISSUE THREAD

**Issue by aaomidi** [microsoft/azuredatastudio#7458]
ExtensionContext is undefined when doing unit tests.
```

	private createSwitchButton(view: azdata.ModelView): void {
		this.switchButton = view.modelBuilder.button().withProperties({
			label: localize('schemaCompare.switchDirectionButton', 'Switch direction'),
			iconPath: {
				light: path.join(this.extensionContext.extensionPath, 'media', 'switch-directions.svg'),
				dark: path.join(this.extensionContext.extensionPath, 'media', 'switch-directions-inverse.svg')
			},
			title: localize('schemaCompare.switchButtonTitle', 'Switch source and target')
		}).component();

```

**Comment by aaomidi** [comment]
Feel free to merge #7460 once this is fixed

**Comment by udeeshagautam** [comment]
Thanks @aaomidi. Will fix and update.

**Comment by aaomidi** [comment]
Fixed with #7492
